src/API/Management/GenericResource.php

```diff
@@ -29,4 +29,58 @@ public function getApiClient()
     {
         return $this->apiClient;
     }
+
+    /**
+     * Normalize pagination parameters.
+     *
+     * @param array        $params   Original parameters to normalize.
+     * @param null|integer $page     Page number, zero-based.
+     * @param null|integer $per_page Per-page count, zero-based.
+     *
+     * @return array
+     */
+    protected function normalizePagination(array $params, $page = null, $per_page = null)
+    {
+        $pagination = [
+            'page' => isset( $params['page'] ) ? $params['page'] : $page,
+            'per_page' => isset( $params['per_page'] ) ? $params['per_page'] : $per_page,
+        ];
+
+        foreach (array_keys( $pagination ) as $pagination_key) {
+            if (empty( $pagination[$pagination_key] )) {
+                unset( $pagination[$pagination_key] );
+                continue;
+            }
+
+            $pagination[$pagination_key] = abs( intval( $pagination[$pagination_key] ) );
+        }
+
+        return array_merge( $params, $pagination );
```

Comment:
```suggestion
        return array_merge( $params, 
                    array_map( function( $i ){ 
                        return abs( intval( $i ) ); 
                    },  array_filter( $pagination )) );
```
But maybe too much functional.

Reply:
I like this better ... TIL that an empty `Callable` in `array_filter` removes false-y values! Going to parse it out into separate lines but thank you for this suggestion.